### PR TITLE
Reload saved variants after new samples loaded

### PR DESCRIPTION
**Pull request by hanars**
After new data is loaded, seqr allele frequencies (plus possibly other annotations) will be updated for all variants of the data type. This will cause the saved variants to be updated to stay in sync with the latest annotations

### CODE REVIEW

**Comment by hanars** [comment]
Realized I missed an edge case on this, will resubmit once functionality is accurate

File: seqr/views/utils/variant_utils.py [hunk 1]
```diff
@@ -27,13 +29,53 @@
...
-    if family_id:
-        saved_variants = saved_variants.filter(family__family_id=family_id)
+def update_projects_saved_variant_json(projects, user_email, **kwargs):
+    success = {}
+    skipped = {}
+    error = {}
+    logger.info(f'Reloading saved variants in {len(projects)} projects')
+    for project_id, project_name, family_ids in tqdm(projects, unit=' project'):
+        try:
+            updated_saved_variant_guids = update_project_saved_variant_json(
```
Comment (bpblanken) [inline seqr/views/utils/variant_utils.py:39]: Just to make sure I've followed this correctly, this makes a single search call to the hail search service per project, with all of the saved variants embedded in the POST body?

Comment (hanars) [reply seqr/views/utils/variant_utils.py:39]: yes, although if theres more than 1000 variants it does them in batches

Comment (hanars) [reply seqr/views/utils/variant_utils.py:39]: There is a better way to do it but it involves some changes to the hail backend, so I was going ot do that as a separate PR: https://github.com/broadinstitute/seqr-private/issues/1362#issuecomment-1971344093

**Review by bpblanken — APPROVED** [review]

**Pull request merged**